### PR TITLE
Update readme and max tested version

### PR DESCRIPTION
**Pull request by duncanp-sonar**
Updated documentation SonarQube version compatibility.

No code changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### License
 
-Copyright 2015-2021 SonarSource.
+Copyright 2015-2022 SonarSource.
```
Comment (duncanp-sonar) [inline README.md:5]: This prompted me to look at the copyright notices in the other files, and they are all out of date. I've created a separate PR for them.

File: RoslynPluginGenerator/UIResources.resx [hunk 2]
```diff
@@ -260,7 +260,7 @@ The conflict will be ignored.</value>
     <value>Maximum supported Roslyn version:</value>
   </data>
   <data name="Program_SupportedSonarQubeVersions" xml:space="preserve">
-    <value>Minimum supported SonarQube version: 7.9.6 (highest version tested against: 9.1)</value>
+    <value>Minimum supported SonarQube version: 7.9.6 (highest version tested against: 9.7)</value>
```
Comment (duncanp-sonar) [inline RoslynPluginGenerator/UIResources.resx:263]: SonarQube v9.7 was released today, so I tested against it and everything still works as expected.

Comment (duncanp-sonar) [reply RoslynPluginGenerator/UIResources.resx:263]: This message is displayed in the console on startup e.g. 
image

(FYI the version numbers for the plugin generator and support Roslyn version are extract programmatically from the relevant assemblies)

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**